### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by egillespie** [egillespie/codelab-sinatra-erb#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](https://cryptic-journey-67221.herokuapp.com/) and meets the following [project](http://lansingcodelab.com/lessons/sinatra-erb/1) criteria:

- [x] The dating profile is not necessarily for humans - questions could be catered to pets, plants, aliens, or members of a fictional universe
- [x] The root of your site (`/`) has a form to collect information that will be embedded in the dating profile
- [x] Submitting the form at the root of the site creates a new dating profile at `/profile`
- [x] One of the form inputs takes the URL of an image to be displayed on the generated profile
- [x] Any CSS or JavaScript is embedded in the page with a `style` or `script` element (we'll learn about including other files in a later lesson)
- [x] For an added challenge, use [the free Formspree service](http://formspree.io/) to generate a contact form on the profile page, so that interested parties have a way of getting in touch with the creator of the profile

**Comment by chrisvfritz** [comment]
Very nice - I literally guffawed! :smiley: It even alarmed the cat. :smirk_cat: Everything seems to work, but I do see a couple minor areas for improvement:

### Matching `label` `for` attributes to `input`/`textarea` `id`s

A neat trick with the `label` element is that if you match the value of the `for` attribute with the `id` of another element, clicking on the label will give the `id`ed element focus. So in this example below:

``` html
<label for="username">Username:</label>
<input type="text" id="username" name="username">
```

Clicking on the `label` with "Username:" will focus the `input` below it, so that the blinking cursor appears and users can start typing right away.

### Sending emails directly to profile creators

Instead of sending an email to _you_ at `erik.gillespie@gmail.com`, see if you can collect an email address from the user, then send the form to them directly by injecting it in the formspree form.

**Comment by egillespie** [comment]
Done and done. Now 20% juicier!

I also specified Ruby 2.2.4 because I got an email that my `codelab-sinatra-intro` project was using the soon-to-be-sunsetted Ruby 2.0.0 and I wasn't sure how to determine which version of Ruby Heroku was using on my other projects where I didn't specify (other than explicitly changing the version and pushing to Heroku to see whether it was changing the Ruby version).

I'm not sure what caused this difference, but Heroku used 2.0.0 for `codelab-sinatra-intro` and 2.2.4 for `codelab-sinatra-erb` until I explicitly specified 2.2.4.
